docs/real-world-examples/ccfraud.md

```diff
@@ -0,0 +1,24 @@
+# Finance example for Federated Learning in Azure ML
```

Comment:
I saw David implemented those different type NNs in the models.py at traininsilo component, but I agree with Thomas that, in addition to the code there, it might be valuable to add a very brief mention about that in the doc here, and also maybe added some minor extra comments/explanations in the models.py file (which currently mainly just codes), adding 1-2 sentence there for each type NN in models.py might probably good enough.

Reply:
@ShizeSu06 - added in this file at very bottom :)